contracts/BSCValidatorSet.sol

```diff
@@ -110,13 +121,17 @@ contract BSCValidatorSet is IBSCValidatorSet, System, IParamSubscriber, IApplica
     if (validatorExtraSet.length == 0) {
       ValidatorExtra memory validatorExtra;
       // init validatorExtraSet
-      uint256 validatorsNum = currentValidatorSet.length;
-      for (uint i; i < validatorsNum; ++i) {
+      for (uint i = 0; i < currentValidatorSet.length; i++) {
```

Comment:
I think the conflict is been false resolved here. It is have been optimized here.

Reply:
Fixed